numpy/linalg/linalg.py

```diff
@@ -2112,6 +2112,9 @@ def norm(x, ord=None, axis=None, keepdims=False):
     """
     x = asarray(x)
 
+    if not issubclass(x.dtype.dtype, floating):
```

Comment:
This looks like a copy-and-double-paste typo. The tests are failing here. Did you mean `x.dtype`?